temporalcli/commandsmd/commands.md

```diff
@@ -2521,12 +2534,40 @@ temporal workflow trace \
 
 Includes options set for [workflow reference](#options-set-for-workflow-reference).
 
-### temporal workflow update: Start and wait for Updates (Experimental)
+### temporal workflow update: Updates (Experimental)
 An Update is a synchronous call to a Workflow Execution that can change its
 state, control its flow, and return a result.
 
 Experimental.
 
+#### Options set for update targeting
+* `--workflow-id`, `-w` (string) -
+  Workflow ID.
+  Required.
+* `--update-id` (string) -
```

Comment:
This should be required for `describe` (and therefore reused for `result` and make `start` be the unnamed set that has a UUID default)